tests/test_form_data.py

```diff
@@ -0,0 +1,16 @@
+import json
+import os
+from .constants import OK_RESPONSE_CODE
+import pytest
+
+
+
+def test_submit_data(client):
+    directory = 'tests/books'
```

Comment:
Maybe make `directory` a constant outside of this function.

Reply:
Done.